pennylane_lightning/core/src/simulators/lightning_gpu/algorithms/tests/Test_AdjointJacobianGPU.cpp

```diff
@@ -0,0 +1,619 @@
+#include <algorithm>
+#include <cmath>
+#include <complex>
+#include <iostream>
+#include <limits>
+#include <type_traits>
+#include <utility>
+#include <variant>
```

Comment:
Can this be removed if it isn't being used?

Reply:
Sure! Removed!